plugin/src/test/groovy/org/openrewrite/gradle/RewritePluginTest.groovy

```diff
@@ -75,8 +75,31 @@ class RewritePluginTest extends RewriteTestBase {
             }
             """.stripIndent()
 
-    @Ignore
-    def "rewriteWarn task will run as part of a normal Java Build"() {
+    def "rewriteWarn task does not have dependsOn link to check task"() {
+        given:
+        projectDir.newFile("settings.gradle")
+        File rewriteYaml = projectDir.newFile("rewrite-config.yml")
+        rewriteYaml.text = rewriteYamlText
+
+        File buildGradleFile = projectDir.newFile("build.gradle")
+        buildGradleFile.text = buildGradleFileText
+
+        when:
+        def result = gradleRunner(gradleVersion, "check").build()
+
+        def rewriteWarnResult = result.task(":rewriteWarn")
+        def checkResult = result.task(":check")
+
+        then:
+        checkResult.outcome == TaskOutcome.UP_TO_DATE
+        // "rewriteWarn" task should not have been called
+        rewriteWarnResult == null
+
+        where:
+        gradleVersion << GRADLE_VERSIONS_UNDER_TEST
+    }
```

Comment:
I don't know if it makes any sense to assert the _absence_ of a dependency relationship... I'd probably just delete this test altogether. The original version of this test asserting the presence of a dependency relationship was also of marginal value

Reply:
Agreed, removed. Although, I do think the original version of the test which asserted the presence of a dependency relationship was a good idea. If for no other reason than for the test to demonstrate the relationship's existence at all in a provable test format.

Ehhh but in any case, yeah, removed 👍  😉